engine/src/cython/initialize.cpp

```diff
@@ -173,7 +173,7 @@ void initialize(int ralId,
 	std::string env_cuda_device_str = env_cuda_device == nullptr ? "" : std::string(env_cuda_device);
 	initLogMsg = initLogMsg + "CUDA_VISIBLE_DEVICES is set to: " + env_cuda_device_str + ", ";
 	
-	size_t buffers_size = 78643200;  // 75 MBs        0.1 * free_gpu_mem_size;
+	size_t buffers_size = 78643200 / 8;  // 75 MBs / 8       0.1 * free_gpu_mem_size;
```

Comment:
We are going to change this value, we need to change it everywhere in the code that we refer to the defaults of TRANSPORT_BUFFER_BYTE_SIZE

Reply:
sure in all honesty we need to set the default to a smaller buffer given the results of our experiments on allocation size